packages/genesys-spark-components/src/components/stable/gux-modal/gux-modal.tsx

```diff
@@ -31,28 +32,52 @@ export class GuxModal {
   @Prop()
   size: GuxModalSize = 'dynamic';
 
+  /**
+   * Indicates/sets whether or not the modal is open. On a native dialog, you should not toggle the
+   * open attribute, due to the unusual behaviors described [here](https://html.spec.whatwg.org/multipage/interactive-elements.html#attr-dialog-open)
+   * In this component, it is safe as this property acts as a proxy for calls to `showModal` and `close`.
+   */
+  @Prop({ mutable: true })
+  open: boolean = false;
+
```

Comment:
This was something I debated when adding the property. Initially we discussed not recommending it as an approach to changing modal state, but after looking up the root cause of the recommendations for not using the native `open` attribute, there wasn't anything to justify preferring one over the other for `gux-modal`.  So I just left it undocumented for now. 

Thinking about it more, since there doesn't seem to be a downside to using it, I don't see much reason to keep it out of the migration guide. Although if it's there, it should probably also be on the examples page somehow. I'll add both of those and we can see how it looks.